app/frontend/src/features/search/SearchBox.tsx

```diff
@@ -35,13 +38,22 @@ export default function SearchBox() {
   return (
     <>
       <form onSubmit={onSubmit} className={classes.root}>
-        <TextField
-          className={classes.box}
-          id="search-query"
-          name="query"
-          label="Search"
-          inputRef={register}
-        ></TextField>
```

Comment:
just use this with variant="standard"

Reply:
It doesn't allow adding the little magnifying glass, I don't think? That's why I had to do the form control stuff